client/validators/userInputSchemas.ts

```diff
@@ -19,5 +21,16 @@ const userInputValidationSchema = z.object({
     }),
 });
 
-export type UserInputValidation = z.infer<typeof userInputValidationSchema>;
-export default userInputValidationSchema;
+// Validation schema for input title
+export const userInputTitleSchema = z.object({
+  title: z
+    .string()
+    .min(1, 'Input title is required')
+    .max(15, {
+      message: 'Title must be less than 15 characters in length!',
+    })
```

Comment:
15 characters might be a little unnecessarily restrictive. Some of the existing prompts are over this - for example, 'teach a dog to sit' is 18 characters. It would probably be difficult to summarize complex topics in so few characters.

Reply:
Thanks for noting this. The restriction is for the input title - which is a temp solution at the moment and the 15 characters is to ensure the whole title can be viewed on the navbar. There is another ticket to enhance the title display.